### ISSUE TITLE
I need ProtocolLib cpu usage optimization?

### ISSUE THREAD

**Issue by Uaizi** [dmulloy2/ProtocolLib#3276]
- [ ] This issue is not solved in a development build

**Describe the bug**
cpu usage increasing unduly

**To Reproduce**

**Expected behavior**
My server seems to be fine, but above 40 players it has become impossible, ProtocolLib started to increase CPU usage in a surreal way, I can't understand why or identify the problem.

**Screenshots**
[link to spark](https://spark.lucko.me/yGKWVONi3P)

**Version Info**
[Provide your ProtocolLib install info with `/protocol dump` through pastebin.](https://nopaste.net/cAe44S231X)

**Additional context**
None, I'm completely lost and confused.

**Comment by Uaizi** [comment]
https://spark.lucko.me/m6AFF0F95D

**Comment by lukalt** [comment]
Hi,

after reviewing the spark profile, I do not believe ProtocolLib is causing the performance problems you are observing.
For ProtocolLib, it mainly reports Epoll wakeup calls, which are not part of ProtocolLib itself but are associated with ProtocolLib due to the way ProtocoLib hooks into the server.

**Comment by Ingrim4** [comment]
I agree with @lukalt, since ProtocolLib is used by multiple plugins it's unlikely that ProtocolLib is at fault. Maybe try removing some of the heavier plugins like RealisticSeasons that use ProtocolLib work with chunks as the MSPT seem to roughly correlate to your increase in chunks.
![{3683837C-F1B7-4D7F-9F73-2C6A520F83A5}](https://github.com/user-attachments/assets/04ecfc09-501c-4343-9f83-2de8577021d0)


Another thing I noticed is high command block usage about 5% per tick.
![{0D93EC43-2A73-43A0-951D-68A6F96920B6}](https://github.com/user-attachments/assets/8e343391-fcbf-44d8-8eb3-1827bb17f7cc)
